include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -664,7 +664,7 @@ __parallel_transform_scan_single_group(oneapi::dpl::__internal::__device_backend
     // Although we do not actually need result storage in this case, we need to construct
     // a placeholder here to match the return type of the non-single-work-group implementation
     using __result_and_scratch_storage_t = __result_and_scratch_storage<_ExecutionPolicy, _ValueType>;
-    __result_and_scratch_storage_t __dummy_result_and_scratch{__exec, 0};
+    __result_and_scratch_storage_t __dummy_result_and_scratch{__exec, 0, 0};
```

Comment:
If we apply https://github.com/oneapi-src/oneDPL/pull/1773/files#r1711563149,
we able to rewrite this as `__result_and_scratch_storage_t __dummy_result_and_scratch{__exec};`

Reply:
See above.